### PR TITLE
Sitepermisssions video and audio combined

### PR DESCRIPTION
**Pull request by Amejia481**
[Demo 🍨 ](https://drive.google.com/file/d/1XDUEhSLS5uweu7PqxeVPAI_hBWvsZv3u/view?usp=sharing)

### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Changelog**: This PR includes a changelog entry or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### CODE REVIEW

File: components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/SitePermissionsFeature.kt [hunk 1]
```diff
@@ -119,109 +120,127 @@ class SitePermissionsFeature(
         session: Session,
         permissionRequest: PermissionRequest
     ): DoorhangerPrompt {
-        return if (!permissionRequest.containsVideoAndAudioSources()) {
-            val prompt = handlingSingleContentPermissions(session, permissionRequest)
-            prompt
+        val context = anchorView.context
+        val uri = Uri.parse(permissionRequest.uri ?: "")
+        val host = " ${uri.host}"
```
Comment (pocmo) [inline components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/SitePermissionsFeature.kt]: nit: There's `toUri()` now :)

e.g.:

```kotlin
val host = permissionRequest.uri?.toUri()?.host ?: ""
```

Comment (pocmo) [inline components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/SitePermissionsFeature.kt]: Why do we need the space in front of the host? 🤔

Comment (Amejia481) [reply components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/SitePermissionsFeature.kt]: Awesome 👍

Comment (Amejia481) [reply components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/SitePermissionsFeature.kt]: Good catch, I didn't notice it!

**Pull request merged**